CMake/cdat_modules/cython_external.cmake

```diff
@@ -10,7 +10,7 @@ ExternalProject_Add(Cython
   BUILD_IN_SOURCE 1
   CONFIGURE_COMMAND ""
   BUILD_COMMAND ${PYTHON_EXECUTABLE} setup.py build
-  INSTALL_COMMAND ${PYTHON_EXECUTABLE} setup.py install ${PYTHON_EXTRA_PREFIX}
+  INSTALL_COMMAND ${PYTHON_EXECUTABLE} setup.py install --old-and-unmanageable ${PYTHON_EXTRA_PREFIX}
```

Comment:
what does --old-and-unmanageable do? It would be nice if you can add documentation here why we added this so that we can remove it in the future.